Add files via upload

Meiger00 opened the pull request:
README for the final submission

Meiger00 committed:
Add files via upload

Meiger00 committed:
Add files via upload

README for the final submission

rcody8 committed:
jcody.ReadMeTxt

- update backend parts of Michael's readme

mae5357 committed:
Update the `readme.md` to match `readme.txt`

mae5357 reviewed (approved):
@Meiger00 I updated the readme.md to match your .txt. If you think that anything should be in the markdown that's not in the text (or vis versa) feel free to change. Otherwise good to merge.

mae5357 commented:
ah didn't see justin's pr #6. reverting in favor of that.

mae5357 commented:
@rcody8 do you think you can add your readme.md from #6  on this branch and we can merge it all as one?

rcody8 committed:
Meiger00-patch-1

- remove duplicate huffpost.csv (readme discusses how this was converted to json
and original can be pulled from kaggle)
- add README-Data.md to provide details on python code/pipelines
- update comments in sandbox.py to match readme-data

mae5357 reviewed (approved):
lgtm
